.gitattributes

```diff
@@ -0,0 +1,4 @@
+.git* export-ignore
+.circleci export-ignore
+.drone* export-ignore
```

Comment:
Can we just remove the .drone files instead? I though they were not being used.

Reply:
Remove .drone and updated .gitattrubutes.